src/main/resources/static/PasswordValidation.js

```diff
@@ -0,0 +1,27 @@
+function validatePassword(){
+    var passwordIn = document.getElementById("password");
```

Comment:
as it is used multiple times -> please make the string "password" a constant. same goes for "passwordConfirm"